lib/machSize.mli

```diff
@@ -32,8 +32,11 @@ val get_off : sz -> sz -> int list
 (* All valid offsets for sz2 in sz1, reduced list *)
 val get_off_reduced : sz -> sz -> int list
 
-(* Smaller of two *)
 val compare : sz -> sz -> int
+
+(* Smaller of two *)
+val smaller_equal : sz -> sz -> bool
```

Comment:
Can this be named `smaller_or_equal` or `less_than_or_equal`?

Reply:
Good catch, thanks.